src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+}
+
+.logo {
+  padding: 10px 0;
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item:not(:first-child) {
+  display: block;
+  margin-left: 20px;
+}
+
+.nav__link {
+  /* white-space: nowrap; */
+  text-transform: uppercase;
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  color: #000;
+  text-decoration: none;
+  position: relative;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+/*  :active*/
+.is-active {
+  text-transform: uppercase;
```

Comment:
This element already has these properties:
```
  display: block;
  height: 60px;
  line-height: 60px;
  text-decoration: none;
  position: relative;
```
Because of `nav__link`. So you should remove the properties above

After that you need to merge styles `color: #00acdc;`. After that, as you can see `is-active` and `nav__link:hover` have the same color. Let's merge these styles